modules/apps/style-book/style-book-service/service.xml

```diff
@@ -25,6 +25,8 @@
 
 		<column name="defaultStyleBookEntry" type="boolean" />
 		<column name="frontendTokensValues" type="String" />
+		<column name="frontendTokenDefinitionId" type="String" />
+		<column name="frontendTokenDefinitionName" type="String" />
```

Comment:
@SelenaAungst @algobob @thiago-buarqque 

Hey everyone... naming these columns is likely the hardest part of this PR. Currently a frontendTokenDefinition does not have an ID or a Name. Rather, a frontendTokenDefinition is provided by a theme or themeCSS client extension that has a name and some sort of unique identifier. A theme has a unique name or id, but a themeCSS client extension has an external reference code. 

So.... what names would make the most sense for these columns? Are there any other details we should consider? Are there any patterns in portal that relate to this?

Reply:
@ethib137 good catch. Should we have a `themeId` and a `externalReferenceCode` columns instead of a unique `id` column? Keep these two columns would make easier to find out if it relates to a theme or CET :thinking: 

About the `name` column, what be better if we called `basedOn`?

Reply:
Thanks for bringing this up @ethib137, so to clarify a theme has a name and id but themeCSS client extension only has an external reference code and no name value? In this case should we rename the id field to fit both, or do we need to add an additional field for the external reference code? Sorry for the confusion

Reply:
Sorry, I didn't mean to imply that. themeCSS client extensions have a name also.

Reply:
I think we need to do some more research to see if there is a similar pattern anywhere else in Portal.

Reply:
Got it, I will look into this today. Thanks again!

Reply:
I see some other components using externalReferenceCodes but I am not sure if I see any similar patterns, please let me know if you have any better suggestions for naming here and I will send a new PR with that. I will keep looking into this tomorrow as well.

Reply:
@algobob @thiago-buarqque @ethib137 Any advice on how to proceed with this?

Reply:
Sorry guys, I missed the email for this

So, I think `themeId` and `themeName` is fine. It's short, fits in the context, and can help find other informations regarding the theme for future functionalities.

The FrontendTokenDefinition is stored based on its theme's id (see [this](https://github.com/liferay/liferay-portal/blob/168590400e29ca523d2ecc3f8488c8aec40e1f8f/modules/apps/frontend-token/frontend-token-definition-impl/src/main/java/com/liferay/frontend/token/definition/internal/FrontendTokenDefinitionRegistryImpl.java#L231) and [this](https://github.com/liferay/liferay-portal/blob/168590400e29ca523d2ecc3f8488c8aec40e1f8f/modules/apps/frontend-token/frontend-token-definition-impl/src/main/java/com/liferay/frontend/token/definition/internal/FrontendTokenDefinitionRegistryImpl.java#L340)), so it makes sense.

What do you guys think?

Reply:
Which, in fact, is the themeId [we pass in to the FrontendTokenDefinition](https://github.com/liferay/liferay-portal/blob/168590400e29ca523d2ecc3f8488c8aec40e1f8f/modules/apps/frontend-token/frontend-token-definition-impl/src/main/java/com/liferay/frontend/token/definition/internal/FrontendTokenDefinitionRegistryImpl.java#L231). So we are already saying that the themeCSS ERC is the themeId for a given frontend token definition

Reply:
That looks good to me

Reply:
Okay... since there is a precedence for it, I think that's okay. It still makes me a little uncomfortable, but I guess I just need to get over that. 😂

Reply:
I started working on the new PR for this, @ethib137 it looks like I do need to run buildService before making some of the changes, as I need to manually add things that rely on the automated methods